src/request/remove-key/index.html

```diff
@@ -73,28 +76,55 @@
d="M18.8 12l-1.2-2.2c-.1-.2-.3-.2-.5-.2h-2.5c-.1 0-.3 0-.4.2L13 12v.5l1.2 2.1c.1.2.3.3.4.3h2.5c.2 0 .4-.1.5-.3l1.2-2.1V12z"/><rect x="20.5" y="10.3" width="14.9" height="3.7" rx="1.9"/><rect x="37.3" y="10.3" width="18.7" height="3.7" rx="1.9"/></g><rect x="26.1" y="19.4" width="17.7" height="1.9" rx=".9"/></g><g opacity=".4" fill="#1F2348"><path fill-rule="evenodd" clip-rule="evenodd" d="M28.2 81.6H20c-.7 0-1.3-.6-1.3-1.3V72c0-.8.6-1.4 1.3-1.4h8.2c.7 0 1.4.6 1.4 1.4v8.2c0 .7-.7 1.3-1.4 1.3zm-6.5-8.1c-.2 0-.3.1-.3.3v4.8c0 .1.1.3.3.3h4.8c.2 0 .3-.2.3-.3v-4.8c0-.2-.1-.3-.3-.3h-4.8zM20 92.5h8.2c.7 0 1.4.6 1.4 1.4v8.1c0 .8-.7 1.4-1.4 1.4H20c-.7 0-1.3-.6-1.3-1.4V94c0-.8.6-1.4 1.3-1.4zm6.5 8.2c.2 0 .3-.2.3-.4v-4.7c0-.2-.1-.4-.3-.4h-4.8c-.2 0-.3.2-.3.4v4.7c0 .2.1.4.3.4h4.8zM41.8 70.7H50c.7 0 1.3.6 1.3 1.4v8.2c0 .7-.6 1.3-1.3 1.3h-8.2c-.7 0-1.4-.6-1.4-1.3V72c0-.8.7-1.4 1.4-1.4zm6.5 8.2c.2 0 .3-.2.3-.3v-4.8c0-.2-.1-.3-.3-.3h-4.8c-.2 0-.3.1-.3.3v4.8c0 .1.1.3.3.3h4.8z"/><path d="M32.3 76.5h1.3a1 1 0 0 0 0-2 .3.3 0 0 1-.3-.4v-2a1 1 0 1 0-2 0v3.4c0 .6.4 1 1 1zM36.4 73.1c.1 0 .3.2.3.4v7.4a1 1 0 1 0 2 0v-8.8c0-.6-.4-1-1-1h-1.3a1 1 0 0 0 0 2zM26.5 85c0 .6.4 1 1 1h4.8c.5 0 1-.4 1-1v-5.4a1 1 0 0 0-2 0v4c0 .3-.2.4-.4.4h-3.4a1 1 0 0 0-1 1z"/><path d="M22.7 84a1 1 0 0 0-1 1v3.4c0 .2-.1.4-.3.4H20a1 1 0 1 0 0 2h17.7c.6 0 1-.4 1-1v-4a1 1 0 0 0-2 0v2.6c0 .2-.2.4-.3.4H24a.3.3 0 0 1-.3-.4V85c0-.5-.5-1-1-1zM37.4 93.9c0-.6-.5-1-1-1h-4.1a1 1 0 0 0-1 1v5.4a1 1 0 1 0 2 0v-4c0-.3.2-.4.3-.4h2.8c.5 0 1-.5 1-1zM50 101H38.4a.3.3 0 0 1-.3-.3V98a1 1 0 1 0-2 0v4c0 .6.4 1 1 1H50a1 1 0 1 0 0-2z"/><path fill-rule="evenodd" clip-rule="evenodd" d="M45.2 98.3h-4a1 1 0 0 1-1.1-1v-4.1c0-.6.5-1 1-1h4.1c.6 0 1 .4 1 1v4c0 .6-.4 1.1-1 1.1zm-2.7-4c-.2 0-.4 0-.4.3v1.3c0 .2.2.4.4.4h1.3c.2 0 .4-.2.4-.4v-1.3c0-.2-.2-.4-.4-.4h-1.3z"/><path d="M49.3 88a1 1 0 0 0-1 1.1V98a1 1 0 1 0 2 0V89c0-.6-.4-1-1-1zM50.3 84.3c0-.5-.4-1-1-1h-7.5a1 1 0 0 0-1 1v4.1a1 1 0 1 0 2 0v-2.7c0-.2.2-.3.4-.3h6c.7 0 1.1-.5 1.1-1z"/></g><g opacity=".3" fill="#1F2348"><path opacity=".7" d="M39 35.3l-1.9-3.4a.8.8 0 0 0-.6-.4h-4c-.2 0-.4.2-.6.4l-2 3.4v.7l2 3.4c.2.2.4.4.7.4h3.9c.2 0 .5-.2.6-.4l2-3.4v-.7z"/><path opacity=".8" d="M39 54.7l-1.9-3.3a.8.8 0 0 0-.6-.4h-4c-.2 0-.4.1-.6.4l-2 3.3v.8l2 3.3c.2.3.4.4.7.4h3.9c.2 0 .5-.1.6-.4l2-3.3v-.8z"/><path opacity=".5" d="M30.6 40.1l-1.9-3.3a.8.8 0 0 0-.7-.4h-3.8c-.3 0-.6.1-.7.4l-2 3.3v.8l2 3.3c.1.3.4.4.7.4H28c.3 0 .6-.1.7-.4l2-3.3v-.8z"/><path opacity=".6" d="M30.6 49.9l-1.9-3.4a.8.8 0 0 0-.7-.4h-3.8c-.3 0-.6.2-.7.4l-2 3.4v.7l2 3.4c.1.2.4.4.7.4H28c.3 0 .6-.2.7-.4l2-3.4V50z"/><path opacity=".8" d="M47.5 40.1l-2-3.3a.8.8 0 0 0-.6-.4H41c-.3 0-.5.1-.7.4L38.4 40c-.1.3-.1.6 0 .8l2 3.3c0 .3.3.4.6.4H45c.3 0 .5-.1.6-.4l2-3.3c.1-.2.1-.5 0-.8z"/><path opacity=".6" d="M47.5 49.9l-2-3.4a.8.8 0 0 0-.6-.4H41c-.3 0-.5.2-.7.4L38.4 50v.7l2 3.4c0 .2.3.4.6.4H45c.3 0 .5-.2.6-.4l2-3.4V50z"/></g><defs><clipPath id="clip0"><path fill="#fff" transform="translate(12.9 9.6)" d="M0 0h44.1v5.3H0z"/></clipPath></defs></svg>
```

Comment:
I would prefer if you would apply proper indentations to elements and add some line breaks to get rid of a 3404 character line.